src/cmds/scripts/win_postinstall.py

```diff
@@ -0,0 +1,632 @@
+        fp.write("PBS_START_MOM=1\r\n")
+        fp.write("PBS_START_SCHED=1\r\n")
+        fp.close()
+        print "pbs.conf file created"
+
+
+def RunPbsmkdirs():
+    os.system('pbs_mkdirs')
+    print 'Securing and recreating pbs_home directories'
+
+
+def InstallingVcredist():
+    print 'Installing Visual C++ Redist..'
+    redist_path = os.path.join(install_path, "vcredist_x86.exe")
+    install_redist = '"' + redist_path + '"' + ' /passive /norestart'
+    if (os.path.exists(redist_path)):
+        os.system(install_redist)
+        print 'Installation done..'
+    else:
+        print "Visual C++ Redist is not available"
+        sys.exit(5)
+
+
+def UpdateDSconfig():
+    postgresql_file_path = os.path.join(
+        pbs_path, "home", "datastore", "postgresql.conf")
+    if (os.path.exists(postgresql_file_path)):
+        # change port to 15007
+        replace(postgresql_file_path, "#port = 5432", "port = 15007")
+        # change listen address to *
+        replace(
+            postgresql_file_path,
+            "#listen_addresses = 'localhost'",
+            "listen_addresses = '*'")
+        # change escape sequence parsing
+        replace(
+            postgresql_file_path,
+            "#standard_conforming_strings = on",
+            "standard_conforming_strings = on")
+        # change logging collector
+        replace(
+            postgresql_file_path,
+            "#logging_collector = off",
+            "logging_collector = on")
+        # change logging directory
+        replace(
+            postgresql_file_path,
+            "#log_directory = 'pg_log'",
+            "log_directory = 'pg_log'")
+        # change logging filename
+        replace(
+            postgresql_file_path,
+            "#log_filename = 'postgresql-%Y-%m-%d_%H%M%S.log'",
+            "log_filename = 'pbs_dataservice_log.%a'")
+        # change logging truncate
+        replace(
+            postgresql_file_path,
+            "#log_truncate_on_rotation = off",
+            "log_truncate_on_rotation = on")
+        # change logging rotation age
+        replace(
+            postgresql_file_path,
+            "#log_rotation_age = 1d",
+            "log_rotation_age = 1440")
+        # change logging format to add timestamp
+        replace(
+            postgresql_file_path,
+            "#log_line_prefix = ''",
+            "log_line_prefix = '%t'")
+        print "postgresql file Successfully configured"
+    else:
+        print "postgresqlfile is not available to configure"
+        exit(7)
+    # Configuring pg_hba.conf file
+    pg_hba_file_path = os.path.join(
+        pbs_path, "home", "datastore", "pg_hba.conf")
+    if (os.path.exists(pg_hba_file_path)):
+        replace(
+            pg_hba_file_path,
+            "# IPv4 local connections:",
+            "# IPv4 connections:")
+        # Change localaddress values for all users
+        original_str = "host    all             all             " + \
```

Comment:
An augmented assignment expression like x += 1 can be rewritten as x = x + 1 to achieve a similar, but not exactly equal effect. In the augmented version, x is only evaluated once. Also, when possible, the actual operation is performed in-place, meaning that rather than creating a new object and assigning that to the target,  the old object is modified instead.
Link : https://docs.python.org/3/reference/simple_stmts.html#augmented-assignment-statements

By said that, These operator is useful while we are updating the variable with new set of values. For normal string concatenation, normal "+" should be fine